### PR TITLE
first correct file for issuemapping, for testing purposes

### PR DESCRIPTION
**Pull request by ErikBorra**
Hi @vecna 

here is the first file with URLs for the country latvia. I have included eu as a separate section and will do so for the other 16 countries as well (this way we don't need to patch the code). Could you merge this pull request so that I can run the script over a Latvian vpn and can get a first visualization for the run?

Thanks,

Erik

### CODE REVIEW

File: special_media/issuemapping-latvia [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+http://www.sam.gov.lv/sm/content/?cat=134
+https://www.em.gov.lv/en/
+
+[health]
+http://www.vm.gov.lv/en/
+http://www.vmnvd.gov.lv/en/about-nhs
+https://www.latvija.lv/epakalpojumi/ep113/apraksts
+http://www.lm.gov.lv/
+
+[security & oversight]
```
Comment (vecna) [inline special_media/issuemapping-latvia]: I'm worried about " & " inside the category name, can you did something with just one word ?

Comment (vecna) [inline special_media/issuemapping-latvia]: ya it is a problem:

      File "/home/qq/T/trackmap/manyutils.py", line 228, in media_file_cleanings
        cleanurl = url_cleaner(line)
      File "/home/qq/T/trackmap/manyutils.py", line 103, in url_cleaner
        raise Exception("Invalid protocol in: %s" % line)
    Exception: Invalid protocol in: [security

the fact is: I can patch it easily, but if these string has to be visualized in html I've to bother also about escaping...

**Comment by ErikBorra** [comment]
Hi @vecna,

i just changed the categories and added extra files. Would you mind taking a look whether it is ok now?

Best,

Erik

**Comment by vecna** [comment]
I'm checking :)

**Comment by vecna** [comment]
Last issue (I'm dealing with): the categories name will be displayed in a website, so I'm putting capital letter, and instead of "[eu]" I'm putting EuropeanUnion, is that ok ?

**Comment by ErikBorra** [comment]
That's fine, I can change it too if you like.

**Comment by ErikBorra** [comment]
ok, awesome, thanks. Merged your changes back in my fork and I'm running austria now.

**Pull request merged**